apps/api/src/app/blueprint/usecases/get-grouped-blueprints/consts.ts

```diff
@@ -1,15 +1,26 @@
 /* eslint-disable max-len */
 /* cSpell:disable */
-import { INotificationTemplate, StepTypeEnum } from '@novu/shared';
+import {
+  DigestTypeEnum,
+  DigestUnitEnum,
+  INotificationTemplate,
+  INotificationTemplateStep,
+  StepTypeEnum,
+} from '@novu/shared';
 
-const STEPS = [
+const STEPS: INotificationTemplateStep[] = [
```

Comment:
I've noticed that after clicking on "Use template" button the Workflow Editor shows some errors on the steps, so I've fixed them here